src/components/Search.module.css

```
@@ -0,0 +1,77 @@
+.searchInputWrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.9);
+  border-radius: 4px;
+  padding: 6px 10px;
+  transition: background-color 0.3s ease;
+}
+
+:global(.dark-theme) .searchInputWrapper {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)